spec/lib/rmt/scc_spec.rb

```diff
@@ -309,7 +309,7 @@ def scc
       before do
         FakeFS.with_fresh do
           FileUtils.mkdir_p(path)
-          File.write(File.join(path, 'organizations_products_scoped.json'), products.to_json)
+          File.write(File.join(path, 'organizations_products.json'), products.to_json)
```

Comment:
Do you mind adding `organizations_products_unscoped.json` and  `organizations_orders.json` here as well?